.github/workflows/nouse_install.yml

```diff
@@ -83,7 +83,7 @@ jobs:
         python3 -c 'import os; print(os.getcwd())'
         python3 -c "import modmesh; print(modmesh.__file__)"
         python3 -c "import _modmesh; print(_modmesh.__file__)"
-        pytest --rootdir=/tmp -v
+        pytest --rootdir=. -v
```

Comment:
Please keep the original code in a comment and add a link to the root cause.  When the bug in pytest is fixed we can change it back.

Reply:
Thank you  for your suggestions. 
I write the comments, including the issue and the original code.